### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by ukff**
Update docs

### CODE REVIEW

File: docs/kyma-environment-broker/03-03-runtime-operations.md [hunk 1]
```diff
@@ -7,7 +7,23 @@ Kyma Environment Broker allows you to configure operations that you can run on a
 
 > **NOTE:** It's important to set lower timeouts for the Kyma installation in the Runtime Provisioner.
 
-## Provisioning
+## Operations
+
```
Comment (piotrmiskiewicz) [inline docs/kyma-environment-broker/03-03-runtime-operations.md:11]: I think we need to describe what stage it is:
 - once stage is finished, won't be repeated (in opposite to step)
 - stage groups steps
do not describe here operation nor steps (it is described at the end of the doc)

File: docs/kyma-environment-broker/03-03-runtime-operations.md [hunk 2]
```diff
@@ -527,3 +527,18 @@ You can configure Runtime operations by providing additional steps. To add a new
 
    </details>
 </div>
+
+## Stages
+
+Operation defines stages and steps, which represents a part of work to be done.
```
Comment (IwonaLanger) [inline docs/kyma-environment-broker/03-03-runtime-operations.md:533]:
```suggestion
An operation defines stages and steps which represent the work you must do.
```

File: docs/kyma-environment-broker/03-03-runtime-operations.md [hunk 3]
```diff
@@ -527,3 +527,18 @@ You can configure Runtime operations by providing additional steps. To add a new
...
+
+The stage is a grouping unit for steps.
```
Comment (IwonaLanger) [inline docs/kyma-environment-broker/03-03-runtime-operations.md:535]:
```suggestion
A stage is a grouping unit for steps.
```

File: docs/kyma-environment-broker/03-03-runtime-operations.md [hunk 4]
```diff
@@ -527,3 +527,18 @@ You can configure Runtime operations by providing additional steps. To add a new
...
+
+Step is a part of the stage
```
Comment (IwonaLanger) [inline docs/kyma-environment-broker/03-03-runtime-operations.md:537]:
```suggestion
A step is a part of a stage.
```

File: docs/kyma-environment-broker/03-03-runtime-operations.md [hunk 5]
```diff
@@ -527,3 +527,18 @@ You can configure Runtime operations by providing additional steps. To add a new
...
+
+Operation can contain multiple stages, and stage can contain multiple steps.
```
Comment (IwonaLanger) [inline docs/kyma-environment-broker/03-03-runtime-operations.md:539]:
```suggestion
An operation can consist of multiple stages, and a stage can consist of multiple steps.
```

File: docs/kyma-environment-broker/03-03-runtime-operations.md [hunk 6]
```diff
@@ -527,3 +527,18 @@ You can configure Runtime operations by providing additional steps. To add a new
...
+
+We group the steps inside one stage when we have some sensitive data which we don't want to store in database, then we are keeping them temporary in memory and pass through steps.
```
Comment (IwonaLanger) [inline docs/kyma-environment-broker/03-03-runtime-operations.md:541]:
```suggestion
You group steps in a stage when you have some sensitive data which you don't want to store in database. In such a case you temporarily store the sensitive data in the memory and go through the steps.
```

File: docs/kyma-environment-broker/03-03-runtime-operations.md [hunk 7]
```diff
@@ -527,3 +527,18 @@ You can configure Runtime operations by providing additional steps. To add a new
...
+
+Once all steps in stage are executed with success, then stage is marked as finished and never repeat again, even if next one will fail.
```
Comment (IwonaLanger) [inline docs/kyma-environment-broker/03-03-runtime-operations.md:543]:
```suggestion
Once all the steps in a stage are successfully executed, the stage is marked as finished and never repeated again, even if the next one fails.
```

File: docs/kyma-environment-broker/03-03-runtime-operations.md [hunk 8]
```diff
@@ -527,3 +527,18 @@ You can configure Runtime operations by providing additional steps. To add a new
...
+If any of steps will fail in given stage, then whole stage will be repeated from beginning.
```
Comment (IwonaLanger) [inline docs/kyma-environment-broker/03-03-runtime-operations.md:544]:
```suggestion
If any steps fail at a given stage, the whole stage is repeated from the beginning.
```

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**